TESTING.md

```diff
@@ -2,27 +2,29 @@
 
 This document aims to provide some basic information on writing and running the `nvim-treesitter` test suite.
 
-## Running tests
+## Setup
 
-### Setup
+### Install plenary and nvim-treesitter dependencies
 
-The only required dependency to run `nvim-treesitter` tests is `plenary.nvim`: https://github.com/nvim-lua/plenary.nvim
+You'll need to have modules `plenary.nvim` and `nvim-treesitter` available on your runtime path.
 
-#### Installing plenary.nvim using packer
+If you use packer, plenary should be on the runtime path already. Otherwise, the simplest way to get it is to add a copy of the `plenary.vim` file to your local `plugin/` directory.
 
-TODO: I'm not sure how to set this up.
+Here's a quick solution to get latest.
 
-#### Installing plenary.nvim using vim-plug
+```
+wget https://raw.githubusercontent.com/nvim-lua/plenary.nvim/master/plugin/plenary.vim -O plugin/plenary.vim
+```
 
-These instrunctions assume you have `vim-plug` configured, and are using it as your package manager. If not, follow the [installation instructions](https://github.com/junegunn/vim-plug#installation) to get started.
+There should be a copy of `nvim-treesitter` in the `plugin/` directory already.
 
-First, install the `plenary.nvim` module if you don't already have it. See the [`vim-plug` usage section](https://github.com/junegunn/vim-plug#usage) for installation instructions.
+### Install and run highlight-assertions 
 
-Next, copy your local version of `plenary.vim` to the `nvim-treesitter/plugin/` directory. You will probably be able to find this file at `~/.local/share/nvim/plugged/plenary.nvim/plugin/plenary.vim`, though it might elsewhere for earlier versions of `vim-plug`.
+The [highlight-assertions](https://github.com/theHamsta/highlight-assertions) project is required for running highlighting tests.
```

Comment:
For running highlight and injection tests, actually, as `highlight-assertions` is the engine to tokenize the test queries

Download the compiled executable through https://github.com/theHamsta/highlight-assertions/releases/

or cargo install like described in that repo